bin/callback.js

```diff
@@ -0,0 +1,84 @@
+const http = require('http')
+
+const CALLBACK_URL = process.env.CALLBACK_URL ? new URL(process.env.CALLBACK_URL) : null
+const CALLBACK_TIMEOUT = process.env.CALLBACK_TIMEOUT || 5000
+const CALLBACK_OBJECTS = process.env.CALLBACK_OBJECTS ? JSON.parse(process.env.CALLBACK_OBJECTS) : {}
+
+/**
+ * @return {Boolean}
+ */
+exports.isCallbackSet = () => {
+  if (CALLBACK_URL) {
+    return true
+  }
+  return false
+}
+
+/**
+ * @param {Uint8Array} update
+ * @param {any} origin
+ * @param {WSSharedDoc} doc
+ */
+exports.callbackHandler = (update, origin, doc) => {
+  const room = doc.name
+  const dataToSend = {
+    room: room,
+    data: {}
+  }
+  const sharedObjectList = Object.keys(CALLBACK_OBJECTS)
+  sharedObjectList.forEach(sharedObjectName => {
+    const sharedObjectType = CALLBACK_OBJECTS[sharedObjectName]
+    dataToSend.data[sharedObjectName] = {
+      type: sharedObjectType,
+      content: getContent(sharedObjectName, sharedObjectType, doc)
```

Comment:
The result of `getContent` is a Yjs type. How do we serialize the Yjs type?

Reply:
Serialized with .toJSON()